### PR TITLE
brightvpn 1.227.590 (new cask)

### PR DESCRIPTION
**Pull request by dmitriis-brd**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by krehel** [comment]
Please attempt to add a zap and a livecheck.

Thanks

File: Casks/b/brightvpn.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+cask "brightvpn" do
+  version "1.227.590"
+  sha256 "e2597911d0c0b7b6b0fba571990a875c7da8fe7d08728512b7f18017478ab881"
+
+  url "https://brightdata.com/static/BrightVPN-1.1-#{version}.dmg",
+      verified: "brightdata.com/static/"
+  name "Bright VPN"
+  desc "Free VPN service"
```
Comment (bevanjkay) [inline Casks/b/brightvpn.rb:8]:
```suggestion
  desc "VPN service"

**Comment by dmitriisbd** [comment]
> Please attempt to add a zap and a livecheck.
> 
> Thanks

Hi, currently we don't support retrieving the latest version outside the website's backend part, so I've left livecheck section empty.

File: Casks/b/brightvpn.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  url "https://brightdata.com/static/BrightVPN-1.1-#{version}.dmg",
+      verified: "brightdata.com/static/"
+  name "Bright VPN"
+  desc "VPN service"
+  homepage "https://brightvpn.com/"
+
+  livecheck do
+    skip "No version information available"
+  end
```
Comment (bevanjkay) [inline Casks/b/brightvpn.rb:13]:
```suggestion
  livecheck do
    url :homepage
    regex(/"vpn_mac"\s*:\s*"v?(\d+(?:[.-]\d+)+)"/i )
  end

File: Casks/b/brightvpn.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+cask "brightvpn" do
+  version "1.227.590"
```
Comment (bevanjkay) [inline Casks/b/brightvpn.rb:2]:
```suggestion
  version "1.1-1.227.590"

File: Casks/b/brightvpn.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  sha256 "e2597911d0c0b7b6b0fba571990a875c7da8fe7d08728512b7f18017478ab881"
+
+  url "https://brightdata.com/static/BrightVPN-1.1-#{version}.dmg",
```
Comment (bevanjkay) [inline Casks/b/brightvpn.rb:5]:
```suggestion
  url "https://brightdata.com/static/BrightVPN-#{version}.dmg",

File: Casks/b/brightvpn.rb (hunk 2 above)
Comment (dmitriisbd) [reply Casks/b/brightvpn.rb:13]: oh, thank you

**Review by bevanjkay — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**